src/installer.js

```diff
@@ -113,38 +113,22 @@ var getSize = function (options, callback) {
 }
 
 /**
- * Determine the dependencies for the `shell.moveItemToTrash` API.
+ * Determine the dependencies for the `shell.moveItemToTrash` API, based on the
+ * Electron version in use.
  */
-var getTrashDepends = function (electronVersion) {
-  if (semver.lt(electronVersion, '1.4.1')) {
-    return 'gvfs-bin'
-  } else if (semver.lt(electronVersion, '1.7.2')) {
-    return 'kde-cli-tools | kde-runtime | trash-cli | gvfs-bin'
-  } else {
-    return 'kde-cli-tools | kde-runtime | trash-cli | libglib2.0-bin | gvfs-bin'
-  }
-}
-
-/**
- * Determine what the default dependencies should be, based on the Electron
- * version in use.
- */
-var getDefaultDepends = function (options, callback) {
+var getTrashDepends = function (options, callback) {
   fs.readFile(path.resolve(options.src, 'version'), (err, tag) => {
     if (err) return callback(err)
 
     // The content of the version file is the tag name, e.g. "v1.8.1"
     var version = tag.toString().slice(1).trim()
-
-    return callback(null, [
-      getTrashDepends(version),
-      'libgconf2-4',
-      'libgtk2.0-0',
-      'libnotify4',
-      'libnss3',
-      'libxtst6',
-      'xdg-utils'
-    ])
+    if (semver.lt(version, '1.4.1')) {
+      return callback(null, 'gvfs-bin')
+    } else if (semver.lt(version, '1.7.2')) {
+      return callback(null, 'kde-cli-tools | kde-runtime | trash-cli | gvfs-bin')
+    } else {
+      return callback(null, 'kde-cli-tools | kde-runtime | trash-cli | libglib2.0-bin | gvfs-bin')
+    }
```

Comment:
> The last commit on this PR made the code a little easier to read regarding default dependencies, in particular, the trash ones.

In my opinion, this makes it more difficult to link to all of the dependency code in one block.

Reply:
I don't really understand why. We need to link to all the dependencies, `depends`, `recommends`, etc. With the new code, I can link to line 165 (from new code), which will have the array for `depends` and all the other dependencies.
Before, we would need multiple links. One for the old `getDefaultDepends` which had the array for `depends`; and another link at line 165 for the other dependencies (`recommends`, `suggests`, etc). Now all the arrays are at line 165

Reply:
Let me commit a change to this branch later tonight to show what I mean.

Reply:
I just realized that I've been using my own fork for this PR, so you won't be able to commit to this particular branch. You can open a new branch (no need for a PR) to show me what you mean.
Also, I think we should use some service (gitter, slack, irc, etc) to chat, and let other people ask questions that shouldn't go under issues

Reply:
> Add more commits by pushing to the dependencies branch on fcastilloec/electron-installer-debian.

I can commit to the branch :)

> Also, I think we should use some service (gitter, slack, irc, etc) to chat, and let other people ask questions that shouldn't go under issues

Out of scope for this PR, but this is what I do for Electron Packager: https://github.com/electron-userland/electron-packager/blob/master/SUPPORT.md (not that people actually read it despite it being linked everywhere)